build.gradle

```diff
@@ -4,22 +4,10 @@ import java.text.SimpleDateFormat
 
 plugins {
   id 'java'
-  id 'maven-publish'
   // https://github.com/johnrengelman/shadow
   id 'com.github.johnrengelman.shadow' version '5.2.0'
-  // plugin to check for dependency updates via task dependencyUpdates
-  // https://github.com/ben-manes/gradle-versions-plugin
-  id 'com.github.ben-manes.versions' version '0.28.0'
```

Comment:
I think we should leave this in even with Dependabot as this will also report on plugin versions and gradle versions where as Dependabot only reports on dependencies themselves

Reply:
I also think it should stay in there

Reply:
I have readded the `ben-manes` plugin